cmd/lightclient/rpc/proto/sentinel.proto

```diff
@@ -2,6 +2,8 @@ syntax = "proto3";
 
 package lightrpc;
 
+option go_package = "../lightrpc";
```

Comment:
you do not need this i think, I will test it out myself.

Reply:
just read your comment above, i will test it

Reply:
thanks